src/riak_cs_riak_client.erl

```diff
@@ -331,6 +331,8 @@ get_user_with_pbc(MasterPbc, Key, true) ->
 get_user_with_pbc(MasterPbc, Key, false) ->
     case strong_get_user_with_pbc(MasterPbc, Key) of
         {ok, _} = OK -> OK;
+        {error, <<"{pr_val_unsatisfied,", _/binary>>} ->
```

Comment:
We can match it when n_val is not 3 (but less than 10).

```erlang
24> F=fun(<<"{pr_val_unsatisfied,", N, ",",M, "}">>) when (M-$0)*2 > (N-$0) -> ok; (_) -> ng end.
#Fun<erl_eval.6.80484245>
25> F(<<"{pr_val_unsatisfied,3,2}">>).                                          
ok               
26> F(<<"{pr_val_unsatisfied,5,3}">>).                                          
ok               
27> F(<<"{pr_val_unsatisfied,3,1}">>).
ng
28> F(<<"{pr_val_unsatisfied,5,2}">>).
ng
```

I think `{3,2}` case is unnecessary and noisy but `{3,1}` case is *NOT*.